### DISCUSSION TITLE
Database connection to Clickhouse

### DISCUSSION THREAD

**Discussion by ravibhure** [hoophq/hoop#513] [Ideas]
Clickhouse is widely used, it would be so nice to provide support to connect to Clickhouse db.

**Comment by andriosrobert** [comment]
Hi, you have two options to use Clickhouse today:
1.  Connect with the Postgres Driver: https://clickhouse.com/docs/en/interfaces/postgresql
2. Create a shell connection with clickhouse CLI and use the web IDE
